### DISCUSSION TITLE
Time out Error naciscdn.org

### DISCUSSION THREAD

**Discussion by mainarabgouveia** [Parcels-code/Parcels#1078] [Q&A]
Hi,

When I try to run fieldset.U.show(), i.e, I get this message "DownloadWarning: Downloading: https://naciscdn.org/naturalearth/110m/physical/ne_110m_coastline.zip
  warnings.warn('Downloading: {}'.format(url), DownloadWarning)".

The website https://naciscdn.org/naturalearth/110m/physical/ne_110m_coastline.zip is offline.

How can I fix it?

Best,

**Comment by erikvansebille** [comment]
This is an Issue with the `cartopy` package, and not something we can fix in Parcels. See their fix at https://github.com/SciTools/cartopy/commit/af814c31829002acd7c1c49ba52cee530c6ac19a and a discussion on backporting at https://github.com/SciTools/cartopy/issues/1834

**Comment by mainarabgouveia** [comment]
Very tks Erik.

Is working well now.
